notebooks/integrations/llama3/rag-elastic-llama3.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
_Commented on notebook `notebooks/integrations/llama3/rag-elastic-llama3.ipynb` Cell 4 Line 1_
```python
!pip install llama-index
```
nitpick: update this to be one line for pip install, similar to how the other notebooks have done this

_Respond and view the context [here](https://gitnotebooks.com/elastic/elasticsearch-labs/pull/266)._

Reply:
Updated the code.